app/models/concerns/reporting/status.rb

```diff
@@ -26,11 +26,13 @@ def started?
     end
 
     def running_for
-      distance_of_time_in_words(started_at, Time.current)
+      # distance_of_time_in_words(started_at, Time.current)
+      ''
     end
 
     def run_time
-      distance_of_time_in_words(started_at, completed_at)
+      # distance_of_time_in_words(started_at, completed_at)
+      ''
```

Comment:
These should be ok to put back under `release-78` (You may need to merge in the base branch)